src/jit/decomposelongs.cpp

```diff
@@ -337,8 +336,6 @@ GenTree* DecomposeLongs::DecomposeLclVar(LIR::Use& use)
     }
     else
     {
-        noway_assert(varDsc->lvLRACandidate == false);
```

Comment:
Yes: LSRA clears this bit on all long-typed locals. This bit is in fact not valid before LSRA runs.